frontend/src/common/unixToFormattedTimeNoSeconds.ts

```diff
@@ -0,0 +1,10 @@
+export const unixToFormattedTimeNoSeconds = (unixDate?: number) => {
+  if (!unixDate) {
+    return '-'
+  }
+  const date = new Date(unixDate * 1000)
+  const hours = '0' + date.getHours()
+  const minutes = '0' + date.getMinutes()
+
+  return hours.substr(-2) + ':' + minutes.substr(-2)
```

Comment:
`.substr()` is deprecated (read [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substr))
also string formatting may look better

```suggestion
  return hours.slice(-2) + ':' + minutes.slice(-2)
```